plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetGalleryFacetImpl.java

```diff
@@ -340,6 +343,13 @@ public void put(final InputStream inputStream) throws IOException, NugetPackageE
         getEventBus().post(new ComponentUpdatedEvent(component, getRepository()));
       }
     }
+
+    if (componentId != null) {
+      try (StorageTx storageTx = openStorageTx()) {
+        maintainAggregateInfo(storageTx, componentId);
+        storageTx.commit();
```

Comment:
Yes, this is temporary; for a reason I don't yet understand the aggregate updates don't 'take' when they're in the same transaction. I suspect some simple but subtle wrinkle somewhere, I'm looking now.